logging/cluster-logging-release-notes.adoc

```diff
@@ -1,5 +1,5 @@
 [id="cluster-logging-release-notes"]
-= Release notes for Red Hat OpenShift Logging
+= Release notes for Red Hat OpenShift Logging 5.2
```

Comment:
Just curious if you have an attributes file for Red Hat OpenShift Logging so that you don't have to type the full product name each time?

Reply:
Not yet. Planned for after this release.

Reply:
@anpingli @jcantrill @igor-karpukhin Please review and approve these release notes for 5.2.